### ISSUE TITLE
Wrong initial trigger state

### ISSUE THREAD

**Issue by vvs-** [opentomb/OpenTomb#94]
In *Atlantis* pillars in room 58 are in the wrong order. If level is loaded in noclip state then they have correct order.

**Comment by Lwmte** [comment]
I remember a bug with flipped rooms which incorrectly set invisible entity attribute to visible (e.g. falling ceiling in Tomb of Qualopec). Possibly whole entity structure is messed up when rooms are flipped.

**Comment by Lwmte** [comment]
Is it still happening?

**Comment by vvs-** [comment]
Yes. Just try *Atlantis* level. No doors are opening in the initial room until you flip it manually.

**Comment by vvs-** [comment]
Seems that the reason for this is simple: flipping doesn't work from initial position. I can only flip that level when Lara has moved from the trigger sector.

**Comment by vvs-** [comment]
The _Atlantis_ initial flip bug happens for several reasons. First, there is a one-shot flag which prevents trigger from working twice. And there is no any entity linked to this trigger at all, just FLIPMAP. Second, it uses flip effect 16, which flips rooms several times back and forth with predefined delay.

**Comment by TeslaRus** [comment]
TR1_MAP14 start: trigger: once = true; **cmd1**: FlipMap[0].mask = 0x1F; flipped to alt; **cmd2**: flicker;
Questions: 
flicker works with flipmap[0] hardcoded?
may be it only flip rooms to back without flipmap routine?

**Comment by vvs-** [comment]
That's a good question. I tested FLICKER_FX on a test map, but didn't try to implement several FLIPMAPs there.

**Comment by Gh0stBlade** [comment]
Can someone please confirm what a "FLIP MAP" is?

**Comment by vvs-** [comment]
FLIPMAP is a trigger command which change flip state for alternate rooms. It's described in TRS3.
